### ISSUE TITLE
profile location in image_creation_and_simulation is wrong

### ISSUE THREAD

**Issue by KrisThielemans** [CCPPETMR/SIRF-Exercises#15]
profiles go through a slice with activity. this would work better
```python
profile_no_attn = acquired_data_no_attn_array[5,20,:]
profile_with_attn = acquired_data_with_attn_array[5,20,:]
profile_attn_factors = attn_factors.as_array()[5,20,:]
```

**Comment by KrisThielemans** [comment]
@rijobro, can you check this?

**Comment by rijobro** [comment]
Current profile looks like this:
![image](https://user-images.githubusercontent.com/33289025/55970525-9a3c0880-5c77-11e9-91a6-71cd126078be.png)

**Comment by KrisThielemans** [comment]
@rijobro, please change to a profile that shows the effect of attenuation more closely.

**Comment by KrisThielemans** [comment]
@rijobro, was this done?

**Comment by rijobro** [comment]
fixed with 6a6a25c1d1aabfac590372658d6b3d755338cf6a
